### DISCUSSION TITLE
How to host Trilium as wiki page but disable the menu to prevent people from editing?

### DISCUSSION THREAD

**Discussion by fliptoback** [TriliumNext/Trilium#4431] [Q&A]
Hi again,

I am trying to replicate the wonderful Trilium Rocks website. I manage to install Trilium on a docker container on a VPS and I can point a custom submain http://wiki.mydomain.net to Trilium. All good.

But in this way Trilium is obviously allowing anyone with access to the url to use it like any server instance. How do I disable the menu button so that people do not simply mess around with the settings, and do things they are not supposed to do?

Edit: I can simply point the submain to the share page - but the share page is missing a lot of features - for instance the site themes (light and dark mode) and all the nice collapsible menu like we have on Trilium Rocks. I really wanted that.

What do I need to do on Trilium side? 

Thanks

**Comment by zerebos** [comment]
Well the domain should definitely be pointing to the share rather than the actual Trilium interface. The extra fanciness of that website is done with some css and js, you can also take advantage of custom templating on 0.62-beta.

**Reply by fliptoback** [reply]
Thanks rauenzi. how do I get 0.62 beta? the latest version on github is 0.61.13. I meant the docker version of it.

**Reply by zerebos** [reply]
For desktop, you just go to the releases page, it's at the top https://github.com/zadam/trilium/releases, for docker you have to update your tag. If you use something other than docker for server, then it depends on your installation.

**Reply by fliptoback** [reply]
Hi Rauenzi, I managed to install v0.62-beta, but I couldnt figure out how to use the custom css template for the share pages.

I tried to copy say the bear-white css theme to the share.css theme and have the shared pages to point to that - it doesnt work.

when you say we can do custom templating, do you not meant the shared pages?

**Reply by zerebos** [reply]
You don't need the beta for just share css, it's for EJS templates which are used to turn the trilium data and turn it into HTML, it's what I am doing for trilium.rocks, but not everyone needs (or should) use this feature. If you're having issues applying a share theme at all, that's a separate issue. Make sure the share theme is also shared itself and that the `~shareCss` property is marked inheritable.

**Reply by fliptoback** [reply]
Thanks Rauenzi. I can actually use the shareCss theme just not any other themes apart from the Frostmiku and the paper one. I was of the (wrong) understanding that the new 0.62 beta now allows the share themes to use any other css themes such as the bear-white css.

Could you please share how do we convert trilium to html using EJS?

I really wanted to do something similar to trilium rocks site - either following your footsteps or if Zadam can make the share css feature to use any other css themes.

Thanks
